scripts/check_configs.ts

```diff
@@ -483,6 +535,10 @@ async function main() {
       // additional ibc token config check
       const isAdditionalTokensConfigValid = isValidAdditionalIbcTokenConfig(jsonData.additional_ibc_token_config, ibcBridgeNames, tokens, network);
       if (!isAdditionalTokensConfigValid) outcomeMap[network] = false;
+
+      // demex trading league config check
+      const isDemexTradingLeagueConfigValid = isValidDemexTradingLeagueConfig(jsonData.demex_trading_league_config, network, marketIds, jsonData.blacklisted_markets, perpPoolIds, tokenSymbols)
+      if (!isDemexTradingLeagueConfigValid) outcomeMap[network] = false;
```

Comment:
As `demex_trading_league_config` is optional, do check if `jsonData. demex_trading_league_config` exists before running the validation function.